frontend/src/concepts/hardwareProfiles/types.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  HARDWARE_PROFILES_DEFAULT_WARNING_MESSAGE,
+  HARDWARE_PROFILES_MISSING_CPU_MEMORY_MESSAGE,
+} from '~/pages/hardwareProfiles/nodeResource/const';
+
+export type WarningNotification = {
+  title: string;
+  message: string;
+};
+
+export enum HardwareProfileWarningType {
+  HARDWARE_PROFILES_DEFAULT_WARNING = HARDWARE_PROFILES_DEFAULT_WARNING_MESSAGE,
+  HARDWARE_PROFILES_MISSING_CPU_MEMORY = HARDWARE_PROFILES_MISSING_CPU_MEMORY_MESSAGE,
+}
```

Comment:
these are just types so use
```suggestion
export enum HardwareProfileWarningType {
  HARDWARE_PROFILES_DEFAULT_WARNING,
  HARDWARE_PROFILES_MISSING_CPU_MEMORY,
}
```